src/System.Binary.Base64/System/Binary/Base64.cs

```diff
@@ -177,30 +196,89 @@ public static int EncodeInPlace(Span<byte> buffer, int sourceLength)
                 var sourceSlice = buffer.Slice(sourceIndex, leftover);
                 var desitnationSlice = buffer.Slice(destinationIndex, 4);
                 destinationIndex -= 4;
-                Encode(sourceSlice, desitnationSlice);
+                TryEncode(sourceSlice, desitnationSlice, out int consumed, out int written);
             }
 
             for (int index = sourceIndex - 3; index>=0; index -= 3) {
                 var sourceSlice = buffer.Slice(index, 3);
                 var desitnationSlice = buffer.Slice(destinationIndex, 4);
                 destinationIndex -= 4;
```

Comment:
I will be changing the function implementation later anyway and aside from changing the Encode call to TryEncode, I haven't touched the "in-place" APIs yet. So, will fix if relevant then.